bot/tests/administration/test_create_key.py

```diff
@@ -11,58 +11,91 @@ class TestAddKeyCommand(BaseTest):
     @pytest.mark.quick
     @pytest.mark.asyncio
     async def test_add_key_valid(self):
-        await self.send_command('/removekey tajny_klucz')
+        key_name = "tajny_klucz"
+        days = 30
+
+        await self.send_command(f'/removekey {key_name}')
         await self.expect_command_result_contains(
-            '/addkey 30 tajny_klucz',
-            [create_key_msg.get_create_key_success_message(30, "tajny_klucz")],
+            f'/addkey {days} {key_name}',
+            [create_key_msg.get_create_key_success_message(days, key_name)],
         )
         await self.expect_command_result_contains(
-            '/removekey tajny_klucz',
-            [remove_key_msg.get_remove_key_success_message("tajny_klucz")],
+            f'/removekey {key_name}',
+            [remove_key_msg.get_remove_key_success_message(key_name)],
```

Comment:
O to mi chodzi z tym czyszczeniem bazy i setup/teardown, teraz musisz uprawiać jakieś gówno z removekey, po to jest setup/teardown

Reply:
Dobra ustaliliśmy że setup teardown jest więc trzeba te zbędne rzeczy powywalać